assets/js/react/components/common/LoginForm/index.jsx

```diff
@@ -0,0 +1,146 @@
+
+import Translate from '../Translate';
+
+const FormWrapper = styled.form`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 0;
+`;
+
+const LoginButton = styled(Button)`
+  display: block;
+  margin: auto;
+`;
+
+const CircularProgressCentered = styled(CircularProgress)`
+  display: block;
+  margin: auto;
+`;
+
+const ErrorText = styled(Typography)`
+  ${({ theme }) => `
+  background-color: ${theme.palette.errorColor};
+  border-radius: 3px;
+  color: ${theme.palette.common.white};
+  padding: 0.5rem;
+  `}
+`;
+
+const LoginForm = ({
+  email,
+  onEmailChange,
+  password,
+  onPasswordChange,
+  onLogin,
+  isFetching,
+  authError,
+}) => {
+  const [isPasswordVisible, setIsPasswordVisible] = React.useState(false);
+
+  const toggleIsPasswordVisible = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+  const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+  };
+
+  const handleEmailChange = (event) => {
+    onEmailChange(event.target.value);
+  };
+  const handlePasswordChange = (event) => {
+    onPasswordChange(event.target.value);
+  };
+
+  return (
+    <>
+      <FormWrapper onSubmit={onLogin}>
+        <FormControl variant="filled">
+          <InputLabel htmlFor="input-with-icon-adornment">
+            <FormattedMessage id="Email" />
+          </InputLabel>
+          <FilledInput
+            name="email"
+            value={email}
+            onChange={handleEmailChange}
+            required
+          />
+        </FormControl>
+
+        <FormControl variant="filled">
+          <InputLabel htmlFor="filled-adornment-password">
+            <FormattedMessage id="Password" />
+          </InputLabel>
+          <FilledInput
+            name="password"
+            type={isPasswordVisible ? 'text' : 'password'}
+            value={password}
+            onChange={handlePasswordChange}
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={toggleIsPasswordVisible}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {isPasswordVisible ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            }
+            required
+          />
+        </FormControl>
+
+        {authError !== '' && (
+          <FormControl>
+            <Fade in={authError !== ''}>
+              <ErrorText>
+                <Translate>{authError}</Translate>
+              </ErrorText>
+            </Fade>
+          </FormControl>
+        )}
+
+        <FormControl>
+          {isFetching ? (
+            <CircularProgressCentered />
+          ) : (
+            <LoginButton type="submit" variant="contained" size="large">
+              <Translate>Log in</Translate>
+            </LoginButton>
+          )}
+        </FormControl>
```

Comment:
Il faudrait peut etre laisser le composant  `Dialog` afficher les bouttons d'actions pour avoir un style identique entre tout les Dialogs ?
Même si actuellement il n'y a pas d'autres dialog...

Reply:
Ce qui m'a gêné avec le mécanisme déjà en place, c'est le fait qu'un seul bouton se retrouve sur la droite. On peut éventuellement rajouter une propriété de centrage facultative au StandardDialog ?

Reply:
C'est une option

Dans le [material design
](https://material.io/components/dialogs#actions) les boutons sont souvent à droites _(même seul)_

Après c'est aussi un choix esthétique. Pour resté cohérent dans l'app il faudrait pe toujours centrer les boutons

Reply:
Yep' je suis d'accord. Va pour à droite, je m'y suis fait. On verra plus tard si y a vraiment besoin d'un centrage ^^

Reply:
Je viens de me rappeler pourquoi j'ai pas voulu utiliser ces boutons... C'est parce que je me suis retrouvé avec mon bouton "Login" en dehors du formulaire qui est dans le content de la fenêtre de dialogue. 

Donc je perds toute la validation automatique du formulaire qui est gérée par le navigateur (+ l'envoi du formulaire avec la touche entrée il me semble) ^^ 
La seule solution que j'ai trouvé actuellement pour dissocier le bouton Log in du formulaire, c'est de faire toute la validation manuellement (checker non-nul et mail valid) avant d'envoyer.